site/content/contribute/server/plugins.md

```diff
@@ -12,17 +12,17 @@ This document covers the plugin infrastructure and how to contribute to it.
 
 ## Building Plugins
 
```

Comment:
Maybe this page should be in the `contribute/plugins` instead of `contribute/server`?

Reply:
This page definitely seems out of place, but I'd like to focus this PR on solving the issue of merging Integrate and Extend. I would like to follow up with a second pass to review all of the pages in the left sidebar to see if they need further restructuring, deprecation, or updates, and this page will almost certainly be included in that.